tsconfig.json

```diff
@@ -15,6 +15,7 @@
     "noEmitOnError": true,
     "noImplicitAny": false,
     "removeComments": false,
+    "resolveJsonModule": true,
```

Comment:
Needed it to import the `tokens.json` file into the Color **Storybook** story and generate the palettes with it.